SpatialGDK/Documentation/content/singleton-actors.md

```diff
@@ -19,7 +19,7 @@ Due to Unreal server-workers spawning their own instances of each Singleton Acto
 
 To set up Singleton Actors for your project, you need to:
 1. Register Singleton Actors by tagging them with the `SpatialType=Singleton` class attribute.
```

Comment:
I _think_ markdown should render this as a properly numbered list. Looks based on the docs like it's missing a newline above the list items, try that instead and see if it works.